node/src/components/rpc_server/rpcs/docs.rs

```diff
@@ -103,7 +103,7 @@ pub trait DocExample {
 
 /// The main schema for the casper node's RPC server, compliant with https://spec.open-rpc.org.
 #[derive(Clone, Serialize, Deserialize, Debug)]
-struct OpenRpcSchema {
+pub struct OpenRpcSchema {
```

Comment:
Do we need this to be `pub` rather than `pub(crate)`?

Reply:
Yes, I tried `pub(crate)` originally